src/components/PostNav.tsx

```diff
@@ -0,0 +1,95 @@
+import blogIcon from "../../public/blog.svg";
+import Image from "next/image";
+import Department from "./Department";
+
+const PostNav = () => {
+	return (
+		<>
+			<div className="navbar bg-base-100 sticky top-0 z-10">
+				<div className="flex-1">
+					<button className="btn btn-square mr-3">
+						<svg
+							fill="#000000"
+							viewBox="0 0 24 24"
+							xmlns="http://www.w3.org/2000/svg"
+						>
+							<g id="SVGRepo_bgCarrier" strokeWidth="0"></g>
+							<g
+								id="SVGRepo_tracerCarrier"
+								strokeLinecap="round"
+								stroke-linejoin="round"
+							></g>
+							<g id="SVGRepo_iconCarrier">
+								<g data-name="Layer 2">
+									<g data-name="collapse">
+										<rect
+											width="24"
+											height="24"
+											transform="rotate(180 12 12)"
+											opacity="0"
+										></rect>
+										<path d="M19 9h-2.58l3.29-3.29a1 1 0 1 0-1.42-1.42L15 7.57V5a1 1 0 0 0-1-1 1 1 0 0 0-1 1v5a1 1 0 0 0 1 1h5a1 1 0 0 0 0-2z"></path>{" "}
+										<path d="M10 13H5a1 1 0 0 0 0 2h2.57l-3.28 3.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0L9 16.42V19a1 1 0 0 0 1 1 1 1 0 0 0 1-1v-5a1 1 0 0 0-1-1z"></path>{" "}
+									</g>
+								</g>
+							</g>
+						</svg>
```

Comment:
❌ 
it is better to put the SVG in its own file, and add that file to the public folder and then you can use it from there, because sometimes it is a lot of text, and no one wants to read an SVG

Reply:
ok